src/components/new-request/search.vue

```diff
@@ -381,6 +419,31 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.scrollTo('namerequest-sbc-header')
   }
 
+  get rules () {
+    return [
+      value => {
+        if (this.isConversion) {
+          const legalType = this.business.legalType
+          this.isPassValidation = legalType === CorpTypeCd.BC_COMPANY ||
+            legalType === CorpTypeCd.BENEFIT_COMPANY ||
+            legalType === CorpTypeCd.BC_ULC_COMPANY
+          return this.isPassValidation || 'This business cannot alter their business type'
+        }
+        return true
+      }]
+  }
+
+  get isAlterOnline (): boolean {
+    if (!this.isConversion) return true
+    return !(this.getConversionType === NrRequestTypeCodes.CONVERT_BEN ||
+      this.getConversionType === NrRequestTypeCodes.CONVERT_CORP ||
+      this.getConversionType === NrRequestTypeCodes.CONVERT_ULBE)
```

Comment:
You already have this logic at existing-request-display.vue::isAllowAlterOnline(). Could it be moved to getters.ts instead?

Reply:
I moved it to common-mixin.ts instead of getters.ts because these two are slightly different. They check different variables.